modules/jt-extending-tekton-using-custom-tasks-and-scripts.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assembly:
+//
+// jenkins-tekton/migrating-from-jenkins-to-tekton.adoc
+
+[id="jt-extending-tekton-using-custom-tasks-and-scripts_{context}"]
+= Extending Tekton using custom tasks and scripts
+
+toc::[]
+
+
+In Tekton, if you don't find the right task in the Tekton Hub or need greater control, you can create custom tasks and scripts to extend its capabilities.
+
+.Example: Custom task for running the `maven test` command
```

Comment:
Good job with documenting all the examples! They all render beautifully in the preview page!